db/comdb2.c

```diff
@@ -1102,8 +1107,7 @@ static void *purge_old_blkseq_thread(void *arg)
     loop = 0;
     sleep(1);
 
-    while (!db_is_stopped()) {
-
+    while (!db_is_stopped() && !db_requests_are_stopped()) {
```

Comment:
Setting requests_stopped would now cause purge_old_blkseq thread to exit forever. I don't think that intended.

Reply:
It exits, but it's restarted in resume_threads.  Same as purge_old_files_thread.  In fact, that needs the same check.